packages/backend/src/managers/TaskRunner.ts

```diff
@@ -48,10 +49,26 @@ export class TaskRunner {
       if (options.errorLabel) {
         task.name = options.errorLabel;
       }
+      if (options.setErrorForSubtasksOnError) {
+        this.setErrorForSubtasksOnError(task, labels);
+      }
       throw err;
     } finally {
       task.progress = undefined;
       this.taskRegistry.updateTask(task);
     }
   }
+
+  private setErrorForSubtasksOnError(task: Task, labels: Record<string, string>): void {
```

Comment:
this is not really a helpful name to understand the purpose

Reply:
1. the parameters: I can see task, and labels, no errors?
2. `set error for sub-task on error` I understand the purpose of the function by reading the code inside it, not the same

Reply:
Some ideas, let's take inspiration from github, `fail-fast` [^1], nice same, easy.

[^1]: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
I don't think you need the first argument, as the task is already in error state, so it will not be in loading state

Reply:
thanks, I've changed with `failFastSubtasks`